qucs/qucs/projectView.cpp

```diff
@@ -157,3 +163,28 @@ ProjectView::exportSchematic()
   }
   return list;
 }
+
+// This function reads the text inside the <description></description> tags from the given file location
+QString ProjectView::ReadDescription(QString file)
+{
+    QFile QucsDocument(file);
+    if (!QucsDocument.open(QIODevice::ReadOnly)) return "";
+    QTextStream in (&QucsDocument);
+    QString line, description;
+    int index, index2;
+    do {
+        line = in.readLine();
+        index = line.indexOf("FrameText0=");
+        if (index != -1) {
+            index2 = line.indexOf(">", index);
+            index+=11;//Skip FrameText0=
+            description = line.mid(index, index2-index);
+            break;
+        }
+    } while (!line.isNull());
+
+    description.replace("\\n", "<br>");
+    QucsDocument.close();
+    if (description == "Title") description = "";//Prevent schematics without description from showing "Title" in the tooltip
```

Comment:
the description string is internationalized: in [schematic.cpp](https://github.com/Qucs/qucs/blob/develop/qucs/qucs/schematic.cpp#L98) the default value of `Frame_Text0` is `tr("Title")` so it would have been better to compare to `tr("Title")`.
Even better will be to show the description only if `showFrame` is set; the rationale is that the description is shown in the schematic and so has probably set to something sensible only if that is set..
This will also handle correctly the case when you open a schematic created with another Qucs language setting. E.g. when Qucs is set to use Italian the `Frame_Text0` is correctly set to *Titolo* but if you now open this schematic from a Qucs with language setting set to English it will check if the description is set to *Title* so the comparison will fail and it will show the *Titolo* string as description in the tooltip.

Reply:
I agree. 
Since this PR is already merged, I've opened a new one (#868) to fix this.